README.md

```diff
@@ -388,3 +388,19 @@ specifically disclaims any liability with respect to this document
 and no contractual obligations are formed either directly or
 indirectly by this document. The technologies, functionality, services,
 and processes described herein are subject to change without notice.
+
+## **Important Notice: This Repository is Deprecated**
```

Comment:
Why is this deprecation **at the very bottom** of the Readme? It even says in the ticket explicitly that it should be ` a disclaimer with big red "!"at top of their README`!